docs/maintainer-guide/releases.md

```diff
@@ -1,29 +1,29 @@
 # Performing releases
 
 1. Create a [new issue](https://github.com/stylelint/stylelint/issues/new) announcing the planned release, e.g. `Release 8.11.1` and include the [template checklist](#new-release-issue-template).
-2. Locally test `main` in the `stylelint-config-*` shareable config repositories. Install current `main` branch (`npm install stylelint/stylelint#main`) and run tests.
-3. Locally test `main` in the [stylelint/stylelint.io](https://github.com/stylelint/stylelint.io) repository.
-4. Locally test `main` in the [stylelint/stylelint-demo](https://github.com/stylelint/stylelint-demo) repository.
+2. Locally test `main` in the `stylelint-config-*` shareable config repositories. Install the current `main` branch (`npm install stylelint/stylelint#main --save-dev`) and run tests.
+3. Locally test `main` in the [stylelint/stylelint-demo](https://github.com/stylelint/stylelint-demo) repository.
+4. Locally test `main` in the [stylelint/stylelint.io](https://github.com/stylelint/stylelint.io) repository.
```

Comment:
[note] The updates are done in the order of `stylelint-demo` and `stylelint.io`.